spec/integration/action_spec.rb

```diff
@@ -1,31 +1,150 @@
 require 'verbalize/action'
 
 describe Verbalize::Action do
-  describe '.verbalize' do
+  let(:a) { double(divide_by: 20) }
+
+  let(:simple_action) do
+    Class.new do
+      include Verbalize::Action
+
+      input :a, :b, :should_fail
```

Comment:
will change to fail when b==0.  Previously the Action had different behavior, makes more sense to constrain by value of b now.